packages/date-picker/src/DatePicker/components/DateInput/DateInput.tsx

```diff
@@ -0,0 +1,122 @@
+import React from "react"
+import { Icon } from "@kaizen/component-library"
+import classnames from "classnames"
+import {
+  FieldGroup,
+  FieldMessage,
+  Input,
+  InputProps,
+  Label,
+} from "@kaizen/draft-form"
+
+import styles from "./DateInput.scss"
+
+type OmittedInputProps =
+  | "startIconAdornment"
+  | "endIconAdornment"
+  | "inputType"
+  | "inputValue"
+
+export interface validationMessagesProps {
+  error?: string | React.ReactNode
```

Comment:
/nit There's other `string | React.ReactNode` typings in this file that can just be `ReactNode`